packages/dashboard-frontend/src/containers/IdeLoader/index.tsx

```diff
@@ -0,0 +1,472 @@
+/*
+ * Copyright (c) 2018-2021 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+
+import React from 'react';
+import { History } from 'history';
+import { connect, ConnectedProps } from 'react-redux';
+import { CancellablePromise, Cancellation, pseudoCancellable } from 'real-cancellable-promise';
+import { AlertVariant } from '@patternfly/react-core';
+import common from '@eclipse-che/common';
+import { AppState } from '../../store';
+import {
+  selectWorkspaceByUID,
+  selectAllWorkspaces,
+  selectIsLoading,
+  selectLogs,
+} from '../../store/Workspaces/selectors';
+import { selectDevworkspacesEnabled } from '../../store/Workspaces/Settings/selectors';
+import * as WorkspaceStore from '../../store/Workspaces';
+import { IdeLoaderSteps, List, LoaderStep } from '../../components/Loader/Step';
+import { buildSteps } from '../../components/Loader/Step/buildSteps';
+import { IdeLoader } from '../../pages/IdeLoader';
+import { RouteComponentProps } from 'react-router';
+import { Workspace } from '../../services/workspace-adapter';
+import { WorkspaceStatus, DevWorkspaceStatus } from '../../services/helpers/types';
+import { DisposableCollection } from '../../services/helpers/disposable';
+import { delay } from '../../services/helpers/delay';
+import getRandomString from '../../services/helpers/random';
+import { filterErrorLogs } from '../../services/helpers/filterErrorLogs';
+import { IdeIframe } from '../../pages/IdeIframe';
```

Comment:
I think we should not support opening IDE for workspaces V1 anymore(and bring a page with an iframe).

@akurinnoy @ibuziuk  WDYT?

Reply:
yeah, for v1 workspaces we should only support delete and convert flows

Reply:
Nice! I'll drop support for Che7 workspaces.